.github/workflows/pull_request.yml

```diff
@@ -14,6 +14,51 @@ on:
 
 # A workflow run is made up of one or more jobs that can run sequentially or in parallel
 jobs:
+  kind_setup:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out forklift repository
+        uses: actions/checkout@v3
+
+      - name: Checkout forkliftci
```

Comment:
this eventually moves here, right? I do not see a need for separate repo. For now that's ok

Reply:
Either the custom action or move the ci

Reply:
I see the forkliftci-repo as more of an experimental playground which has stuff which we might not need at the end.